cpp/log/etcd_consistent_store-inl.h

```diff
@@ -18,7 +18,7 @@ namespace {
 // etcd path constants.
 const char kClusterConfigFile[] = "/cluster_config";
 const char kUnsequencedDir[] = "/unsequenced/";
```

Comment:
They're not really "unsequenced" there anymore, they're more just... entries, right? Maybe rename this, otherwise it might be a bit confusing down the road?